hydra/core/override_parser/types.py

```diff
@@ -235,6 +236,16 @@ class Override:
     # Configs repo
     config_loader: Optional[ConfigLoader] = None
 
+    def __post_init__(self) -> None:
+        if self.pkg2 is not None:
+            # DEPRECATED: remove in 1.1
+            msg = (
+                "\nSupport for overriding the package via the command line "
+                "is deprecated since Hydra 1.0.5 and will be removed in Hydra 1.1. "
+                "For more details, refer https://github.com/facebookresearch/hydra/issues/1140."
```

Comment:
```suggestion
                "\nSupport for renaming packages via the command line "
                "is deprecated since Hydra 1.0.5 and will be removed in Hydra 1.1. "
                "For more details, see https://github.com/facebookresearch/hydra/issues/1140."
```

Reply:
Will do!